carma/launch/plugins.launch.py

```diff
@@ -451,13 +454,40 @@ def generate_launch_description():
                 ],
                 parameters=[
                     vehicle_characteristics_param_file, #vehicle_response_lag
-                    pure_pursuit_tuning_parameters, #pure_pursuit calibration parameters
                     vehicle_config_param_file
                 ]
             ),
         ]
     )
 
+    carma_trajectory_follower_wrapper_container = ComposableNodeContainer(
+        package='carma_ros2_utils',
+        name='carma_trajectory_follower_wrapper_container',
+        executable='carma_component_container_mt',
+        namespace=GetCurrentNamespace(),
+        composable_node_descriptions=[
+            ComposableNode(
+                    package='trajectory_follower_wrapper',
+                    plugin='trajectory_follower_wrapper::TrajectoryFollowerWrapperNode',
+                    name='trajectory_follower_wrapper',
+                    extra_arguments=[
+                    {'use_intra_process_comms': True},
+                    {'--log-level' : GetLogLevel('trajectory_follower_wrapper', env_log_levels) }
+                ],
+                remappings = [
+                    ("plugin_discovery", [ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''), "/plugin_discovery" ] ),
+                    ("ctrl_raw", [ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''), "/ctrl_raw" ] ),
+                    ("trajectory_follower_wrapper/plan_trajectory", [ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''), "/plugins/pure_pursuit/plan_trajectory" ] ),
```

Comment:
I think we want to remap this to `[ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''), "/plugins/trajectory_follower_wrapper/plan_trajectory" ]` (or similar). The default control plugin topic is defined in `trajectory_executor`'s configuration here: https://github.com/usdot-fhwa-stol/carma-platform/blob/11047f2bde3dc2f37b7d2744cd41a3d9772bbc42/trajectory_executor/config/parameters.yaml#L14